acro/acro_stata_parser.py

```diff
@@ -375,7 +375,7 @@ def run_regression(command: str, data: pd.DataFrame, varlist: list) -> str:
 def get_regr_results(results: sm_iolib_summary.Summary, title: str) -> str:
     """Translates statsmodels.io.summary object into prettified table."""
     res_str = title + "\n"
-    for table in get_summary_dataframes(results.summary().tables):
+    for table in ACRO().get_summary_dataframes(results.summary().tables):
```

Comment:
This smells wrong and would be fixed if the function was removed from the class (see comment above)